rts/Rendering/Units/UnitDrawer.cpp

```diff
@@ -1333,6 +1329,182 @@ bool CUnitDrawerLegacy::ShowUnitBuildSquare(const BuildInfo& buildInfo, const st
 	return canBuild;
 }
 
+static inline bool IsZeroColor(SColor color) {
+	return color.r == 0 && color.g == 0 && color.b == 0 && color.a == 0;
+}
+
+void CUnitDrawerLegacy::AddLuaBuildSquare(const BuildInfo& buildInfo, LuaBuildSquareOptions& opts) {
```

Comment:
Could add a FIXME code comment that this is partially a copypaste of function X from file Y.cpp done due to compilation error Z

Reply:
I think you meant `ParseBuildInfo`, it has been commented